### DISCUSSION TITLE
Issue: No response to target latency after automatic resync with L

### DISCUSSION THREAD

**Discussion by Toastie** [f1multiviewer/issue-tracker#4] [General]
After automatically resyncing a player with L the player wont respond to changes in the target latency anymore. Reloading with CTRL+R fixes this.

**Comment by f1multiviewer** [comment]
I've made some improvements to this, can you try again?

**Comment by jeppeolesen** [comment]
Closing this as stale. If it's still an issue @Toastie please file it under issues: https://github.com/f1multiviewer/issue-tracker/issues
